src/shell/shell_cmds.cpp

```diff
@@ -129,25 +129,39 @@ bool lookup_shell_cmd(std::string name, SHELL_Cmd &shell_cmd)
 	return true;
 }
 
-bool DOS_Shell::CheckConfig(char *cmd_in, char *line) {
+bool DOS_Shell::ExecuteConfigChange(const char* const cmd_in, const char* const line)
+{
+	assert(control);
+	const auto section_dos = static_cast<Section_prop*>(
+	        control->GetSection("dos"));
+	assert(section_dos);
+	if (!section_dos->Get_bool("shell_config_shortcuts")) {
+		return false;
+	}
```

Comment:
Suggest putting this in the `[dosbox]` section because it's not DOS emulation related (yes: these are commands in the shell, but it's not DOS internal behavior - like memory / timing / interrupts / file handling / etc.).